README.md

```diff
@@ -252,6 +252,32 @@ make container
 Simply build and run kube-state-metrics inside a Kubernetes pod which has a
 service account token that has read-only access to the Kubernetes cluster.
 
+### Prometheus Operator
+
+The Prometheus Operator ([`kube-prometheus`](https://github.com/prometheus-operator/kube-prometheus/)) installs kube-state-metrics as one of its [components](https://github.com/prometheus-operator/kube-prometheus#kube-prometheus); you do not need to install kube-state-metrics if you're using the Prometheus Operator.
```

Comment:
Prometheus operator does not install kube-state-metrics, but the kube-promethus stack does, so this is a bit misleading.

Reply:
Is the vernacular that a stack include an Operator?

As someone outside of the core this isn't a differentiation with which I'm familiar.

If I change this to say "The kube-prometheus stack (including the Prometheus Operator) installs..."?

Reply:
the prometheus operator does not install kube-state-metrics and knows nothing about kube-state-metrics itself. kube-prometheus has the manifests and knowledge around it, coming from this projects jsonnet mixin. So mentioning Prometheus operator is misleading unless you are referring to the org? We tend to refer to it as kube-prometheus stack.

Reply:
Fixed: https://github.com/kubernetes/kube-state-metrics/pull/1412/commits/1b44f89f78b73d9b211aa6c69147642e78bca71a